peer/peer.go

```diff
@@ -87,23 +87,23 @@ func (p *Peer) DiscoverPeers(ctx context.Context) {
 }
 
 func (p *Peer) Run(ctx context.Context) {
-	p.transport.Receive(ctx, func(from id.Signatory, msg wire.Msg) error {
+	p.transport.Receive(ctx, func(from id.Signatory, packet wire.Packet) error {
 		// TODO(ross): Think about merging the syncer and the gossiper.
-		if err := p.syncer.DidReceiveMessage(from, msg); err != nil {
+		if err := p.syncer.DidReceiveMessage(from, packet); err != nil {
```

Comment:
The syncer doesn't use the `IPAddr` field of the `Packet`, so the `DidReceiveMessage` function interface doesn't need to change. You may wish to keep the change for consistency, however. I would err on the side of reverting the change.